docs/architecture/c#-client-library.md

```diff
@@ -63,6 +64,17 @@ public record EmissionsForecast
 }
 ```
 
+```c#
+namespace GSF.CarbonAware.Models;
+public record CarbonIntensity
```

Comment:
I think the mental model of them is different. In the `CarbonIntensity` model, have the `Start` and `End` explicitly are important because users ask for the carbon intensity within that range. and they may ask for multiple ranges, so we want the return value to match. If anything, maybe the `EmissionsData` model can inherit from the `CarbonIntensity` model so it has that extra `End` field

Reply:
What I am saying is not that either one is preferred, but that having both is confusing because there is little distinction between them.  Given that the SDK is a greenfield consumer, it might be worthwhile to shed unnecessary baggage.

What would it look like if all emissions methods returned `CarbonIntensity` objects?

Reply:
I see what you mean. Given that the idea of the `CarbonIntensity` object was more for the batch results, I think it's worth just leaving Emissions data now. Eventually we may want to reconsider the start + duration instead of start/end but I agree that can be something we think about while not necessarily picking a side in this PR